### PR TITLE
Adds Internal Recruiter job ad.

### PR DESCRIPTION
**Pull request by gabrielamatias**

### CODE REVIEW

**Comment by gabrielamatias** [comment]
@petehamilton : please review.

File: app/pages/about/jobs/positions/internal-recruiter.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+export default class InternalRecruiter extends React.Component {
+  displayName = 'InternalRecruiter'
+
+  render() {
+    return (
+      <JobsPage>
+        <h3 className='u-text-heading-light u-text-m u-color-heading'>Internal Recruiter</h3>
+        <p className='para'>
+        GoCardless is growing fast. We’ve built a world-class product that opens up Direct Debit
```
Comment (petehamilton) [inline app/pages/about/jobs/positions/internal-recruiter.js]: I'd indent this in one (inside the tags)

File: app/pages/about/jobs/positions/internal-recruiter.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>The role</h4>
+        <p className='para'>
+        We’re looking for an internal recruiter to help us grow our Marketing, Business Development,
+        and Sales departments. As a member of our People team you will be responsible for ensuring all
+        our candidates have a great experience. You will be an ambassador of our employer brand and own
+        the hiring process from application to offer.
+        </p>
+        <p className='para'>
+        On a day-to-day basis you will be:
+       </p>
```
Comment (petehamilton) [inline app/pages/about/jobs/positions/internal-recruiter.js]: Alignment

File: app/pages/about/jobs/positions/internal-recruiter.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        <ul className='list'>
+          <li>
```
Comment (petehamilton) [inline app/pages/about/jobs/positions/internal-recruiter.js]: I'd put these inline personally. Up to you though!

```
<li>Managing candidates through the hiring process.</li>
```
```

**Comment by petehamilton** [comment]
Looks good, few comments on alignment.

**Pull request merged**